app/res/layout/screen_login.xml

```diff
@@ -86,21 +86,19 @@
                                 android:textSize="@dimen/text_medium"
                                 android:visibility="gone"/>
 
-                            <EditText
+                            <AutoCompleteTextView

```

Comment:
I can remove the autocomplete entirely for now if we don't think it's a worthwhile addition; I agree that I don't know how much value it really adds.